### ISSUE TITLE
Launch App only works when Easer is in the foreground

### ISSUE THREAD

**Issue by Redsandro** [renyuneyun/Easer#446]
**Describe the bug**

Launch App on event (e.g. bluetooth connection) doesn't always work. I can't start apps unless Easer is active in the foreground.

Important: I do have the _"Easer running in the background"_ permanent notification. The profile _does_ run. It just fails to launch the app.

![image](https://user-images.githubusercontent.com/1702193/184112910-c259bd5b-3cbd-4ca7-8c23-5ccb741bdc58.png)

Is this an obvious problem where I have the wrong settings on my phone? Or is this a new policy coming from Android 12? Or is this a bug?

**To Reproduce**

![image](https://user-images.githubusercontent.com/1702193/184113016-50ebcef4-7b38-4a14-a8b0-5051c755d1df.png)

**Expected behavior**

- :heavy_check_mark: apps can run with Easer in the foreground
- :heavy_check_mark: apps can run with Easer in the background

**Actual behavior**

- :heavy_check_mark: apps can run with Easer in the foreground
- :x: apps can run with Easer in the background

**Error log:**

![image](https://user-images.githubusercontent.com/1702193/184113097-d18c3dc8-c6b0-421a-a58b-040894226e4c.png)

**Extra phone info:**
 - ROM: LineageOS 19.1 Android 12
 - Easer version: 0.8.2.3

**Comment by OliveiraHermogenes** [comment]
Duplicate of #357?

**Comment by Redsandro** [comment]
@OliveiraHermogenes is it? The linked issue is two years old and two Android versions ago; in the current version of Easer there are settings for foreground service and persistent notification, and I have both enabled. If that still makes this a duplicate let me know and I can close this one.

**Comment by OliveiraHermogenes** [comment]
@Redsandro I honestly don't know. They do seem to be related though, and a fix for one is possibly a fix for the other. My intention however was more to link the two issues for the benefit of people searching around than to advocate for closing this one.

**Comment by Redsandro** [comment]
@OliveiraHermogenes Thank you. I'm not sure either. Given the possible duplicate age, one would suspect Easer to be unmaintained. However, there have been a dozen tagged (bugfix) releases since. So either (and/or):

1. Not many users launch apps/activities with Easer; or
2. not many users run Android 10+; or
3. Easer has very few users; or
4. this is not a duplicate.

**Comment by SpiritCroc** [comment]
I don't see how this would not be a duplicate tbh, sounds like the exact same bug description

**Comment by Redsandro** [comment]
This was when I still assumed the _foreground service_ and _persistent notification_ were a direct result of issues found 2 years ago, because there have been a dozen new tagged releases since.

I assumed that broken _core functions_ such as starting apps or activities would be pretty high on the list of priorities in those tagged release.

I was wrong and I will close the issue.
